src/test/resources/all-map-data-in-variant.json

```diff
@@ -211,16 +259,84 @@
       "q3": 18.0,
       "permanentLimit1": 25.0,
       "permanentLimit3": 54.0,
-      "ratioTapChanger3Position": 2,
-      "phaseTapChanger1Position": 1,
       "phaseTapChanger1": {
         "lowTap": 0,
-        "highTap": 2
+        "highTap": 2,
+        "tapPosition": 1,
+        "targetVoltage": 25.0,
+        "voltageRegulation": true,
+        "targetDeadBand": 22.0,
+        "regulationMode" : "CURRENT_LIMITER",
+        "regulatingValue" : 25.0,
+        "regulatingTerminalConnectableId": "TWT",
+        "regulatingTerminalConnectableType": "THREE_WINDINGS_TRANSFORMER",
+        "regulatingTerminalVlId": "VLHV1",
+        "phaseTapSteps": [{
+          "r": 1,
+          "x": 4,
+          "g": 0.5,
+          "b": 1.5,
+          "rho": 0.99,
+          "alpha": -10.0,
+          "index": 0
+        }, {
+          "r": 1.2,
+          "x": 4.2,
+          "g": 0.7,
+          "b": 1.7,
+          "rho": 1.01,
+          "alpha": 10.0,
+          "index": 2
+        }, {
+          "r": 1.1,
+          "x": 4.1,
+          "g": 0.6,
+          "b": 1.6,
+          "rho": 1.0,
+          "alpha": 0.0,
+          "index": 1
+        }]
       },
       "ratioTapChanger3": {
         "lowTap": 0,
-        "highTap": 2
+        "highTap": 2,
+        "tapPosition": 2,
+        "targetDeadBand": 22.0,
+        "targetVoltage": 220.0,
+        "loadTapChangingCapabilities": false,
+        "voltageRegulation": false,
+        "regulatingTerminalConnectableId": "TWT",
+        "regulatingTerminalConnectableType": "THREE_WINDINGS_TRANSFORMER",
+        "regulatingTerminalVlId": "VLHV1",
+        "ratioTapSteps": [
+          {
+            "r": 1,
+            "b": 1.5,
+            "g": 0.5,
+            "rho": 0.99,
+            "x": 4,
+            "index": 0
+          },
+          {
+            "r": 1.1,
+            "b": 1.6,
+            "g": 0.6,
+            "rho": 1,
+            "x": 4.1,
+            "index": 1
+          },
+          {
+            "r": 1.2,
+            "x": 4.2,
+            "g": 0.7,
+            "b": 1.7,
+            "rho": 1.01,
+            "index": 2
+          }
+        ]
       },
+      "ratioTapChanger3Position": 2,
```

Comment:
same ? it's already set in `ratioTapChanger3.tapPosition`

Reply:
✓